### PR TITLE
Named spacing units and Icons

### LINKED ISSUE #141: Preload images for Secret Key generation animation
So that they don't flicker while loading at each step during the animation, smoothening out the experience.

### LINKED ISSUE #155: Add icon to form error messages

### LINKED ISSUE #257: Add in all icons to ui
I've started adding in all the icons from the two repos into UI, and exporting them there. I still need to update all the imports from the two apps into the new release of ui. I am not sure yet on how the automated publishing is working.

### PR DESCRIPTION
**Pull request by aulneau**

### CODE REVIEW

**Comment by hstove** [comment]
Is this a WIP?

And, how would I use `sizes`? `pt="loose"`, `fontSize="tight"`, etc?

File: packages/ui/src/icons/index.tsx [hunk 1]
```diff
@@ -1 +1,223 @@
-export * from './chevron';
+import React from 'react';
```
Comment (kyranjamie) [inline packages/ui/src/icons/index.tsx]: This file will continue to grow over time — it's already too big imo. It'll be cleaner to give each icon its own file, and export them from the index. What are your thoughts?

Comment (kyranjamie) [reply packages/ui/src/icons/index.tsx]: It also means we can easily jump to the file of a given icon, as it has a descriptive name — not just `index.tsx`.

Comment (kyranjamie) [reply packages/ui/src/icons/index.tsx]: It also means we can easily jump to the file of a given icon, as it has a descriptive name — not just `index.tsx`.

Comment (aulneau) [reply packages/ui/src/icons/index.tsx]: Yep agreed--I'll make them their own files

**Comment by aulneau** [comment]
You would not typically use it for `fontSize`, but you'd use it for things like p, m, etc

**Comment by aulneau** [comment]
@hstove you can see the implementation here: https://github.com/blockstack/ux/pull/258/commits/4b1eff276c5dcd5346f5194a73a1021322d85588

**Review by hstove — APPROVED** [review]

**Comment by hstove** [comment]
@timstackblock , I'm assigning you here to QA this before we merge into production. This is all just refactoring, so we just want to make sure that nothing looks out of place. Specifically, we've refactored how we handle spacing, so if you see any weird margins or padding, then that's probably something to flag.

Staging site: https://deploy-preview-258--authenticator-demo.netlify.com/

@aulneau , mind rebasing and moving this all into just 1 commit? No rush, just before we merge.

**Review by kyranjamie — APPROVED** [review]

**Comment by timstackblock** [comment]
This looks good.

**Comment by hstove** [comment]
Thanks Tim - I'll rebase and merge this

**Pull request merged**